### ISSUE TITLE
Training process killed after epoch 1

### ISSUE THREAD

**Issue by aseshasa** [fizyr/keras-retinanet#637]
I am trying to  train a model on a GCP cloud instance. I am using the model from https://github.com/fizyr/keras-retinanet. I use screen to start the code and run it after logging out. It takes a day to complete the first epoch and then the process outputs : Killed96160. Does the code automatically kill itself after epoch 1? Or is it the instance doing that? If so, how do I fix it?

**Comment by aseshasa** [comment]
It runs continuously when I train it without evaluating the validation set every epoch. Have you experienced this? Also, how many epochs do you recommend for 500 object classes?

**Comment by de-vri-es** [comment]
It might be that the evaluation uses too much resources and triggers something on GCP that kills the instance. But I'm just guessing here, I have no experience with GCP.

**Comment by hgaiser** [comment]
Did you resolve the issue?

**Comment by aseshasa** [comment]
I found out that the issue was caused when the RAM had been exhausted. I resolved it by configuring swap memory.

**Comment by hgaiser** [comment]
Alright, then I'm closing the issue.
